itests/org.openhab.binding.hue.tests/src/main/java/org/openhab/binding/hue/internal/AbstractHueOSGiTestParent.java

```diff
@@ -21,7 +21,7 @@
 /**
  * @author Markus Rathgeb - migrated to plain Java test
  */
-public class AbstractHueOSGiTest extends JavaOSGiTest {
+public class AbstractHueOSGiTestParent extends JavaOSGiTest {
```

Comment:
Just for my personal understanding: The suffix `Parent` prevents JUnit to interpret this class. It's just a random value without any specific meaning, am I right? I saw it in https://github.com/openhab/openhab2-addons/pull/5377 too.

Reply:
Yes it assumes each class ending with Test has tests. But it is mistaken. :wink: